src/amp-story-player/amp-story-player-impl.js

```diff
@@ -305,8 +309,10 @@ export class AmpStoryPlayer {
     iframeEl.classList.add('story-player-iframe');
     this.iframes_.push(iframeEl);
 
-    applySandbox(iframeEl);
-    this.addSandboxFlags_(iframeEl);
+    if (typeof __AMP_VISUAL_TEST === 'undefined') {
```

Comment:
Why do we not want sandbox during visual tests? Shouldn't the iframes have the same permissions?

Reply:
this was just me trying to troubleshoot the issue. I don't think we need to do this in actuality